docs/dev-proc/source-code-structure.md

```diff
@@ -38,4 +38,48 @@ with following exceptions:
   happen it would be better to have it fixed by original author than the person
   that tries to upstream it some time later.
 
-Force-pushes to other branches are unconditionally forbidden.
+Force-pushes to  `<platform>/release` branches are unconditionally forbidden.
+
+## Merging guidelines
+
+We want to keep the history linear. The `rebase` merging strategy is desired.
+Merge commits in the code repositories are not allowed. The `rebase` strategy
+should be the only one available in the GitHub web UI.
+
+Is is, however, **strongly advised not to use GitHub web UI** to perform code
```

Comment:
```suggestion
It is, however, **strongly advised not to use GitHub web UI** to perform code
```

Reply:
@krystian-hebel Can you fix it as well while changing git commands? Thanks!

Reply:
Done